src/EnergyPlus/HeatBalanceIntRadExchange.cc

```diff
@@ -571,11 +569,22 @@ namespace HeatBalanceIntRadExchange {
             // Initialize the surface pointer array
             int enclosureSurfNum = 0;
             for (int const zoneNum : thisEnclosure.ZoneNums) {
+                int priorZoneTotEnclSurfs = enclosureSurfNum;
                 for (int surfNum = Zone(zoneNum).SurfaceFirst, surfNum_end = Zone(zoneNum).SurfaceLast; surfNum <= surfNum_end; ++surfNum) {
                     if (!Surface(surfNum).HeatTransSurf) continue;
                     ++enclosureSurfNum;
                     thisEnclosure.SurfacePtr(enclosureSurfNum) = surfNum;
                 }
+                // Store SurfaceReportNums to maintain original reporting order
```

Comment:
All changes in `HeatBalanceIntRadExhchange` are only to preserve reporting order for view factors.